community/pdns/APKBUILD

```diff
@@ -121,23 +119,24 @@ _mv_backend() {
 		"$subpkgdir/usr/lib/pdns/pdns/"
 }
 
-backend_bind()          { _mv_backend bind; }
-backend_geoip()         { _mv_backend geoip geoip; }
-backend_ldap()          { _mv_backend ldap; }
-backend_lua()           { _mv_backend lua lua; }
-backend_mariadb()       { _backend mariadb $pkgname-backend-mysql; }
-backend_mydns()         { _mv_backend mydns; }
-backend_mysql()         { _mv_backend gmysql mariadb-client; }
-backend_odbc()          { _mv_backend godbc unixodbc; }
-#backend_opendbx()       { _mv_backend opendbx; }
-#backend_oracle()        { _mv_backend goracle; }
-backend_pipe()          { _mv_backend pipe; }
-backend_pgsql()         { _mv_backend gpgsql postgresql-client; }
-backend_random()        { _mv_backend random; }
-backend_remote()        { _mv_backend remote; }
-backend_sqlite3()       { _mv_backend gsqlite3 sqlite; }
-#backend_tinydns()       { _mv_backend tinydns; }
-
-sha512sums="59a7a52468f6daae8de01bb2b08d812906ef58047026369895341cfff253a5b9ba29d6a6b43a822f1632641eec34fa1afa6fbb5b0ba5e72ecce8e61787892136  pdns-4.1.10.tar.bz2
+backend_bind() { _mv_backend bind; }
+backend_geoip() { _mv_backend geoip geoip; }
+backend_ldap() { _mv_backend ldap; }
+backend_lua() { _mv_backend lua lua; }
+backend_mariadb() { _backend mariadb $pkgname-backend-mysql; }
+backend_mydns() { _mv_backend mydns; }
+backend_mysql() { _mv_backend gmysql mariadb-client; }
+backend_odbc() { _mv_backend godbc unixodbc; }
+#backend_opendbx() { _mv_backend opendbx; }
+#backend_oracle() { _mv_backend goracle; }
+backend_pipe() { _mv_backend pipe; }
+backend_pgsql() { _mv_backend gpgsql postgresql-client; }
+backend_random() { _mv_backend random; }
+backend_remote() { _mv_backend remote; }
+backend_sqlite3() { _mv_backend gsqlite3 sqlite; }
+backend_tinydns() { _mv_backend tinydns; }
```

Comment:
If you intend to turn on tinydns backend you will also need to add a subpackage entry to use this function.